Src/CSharpier/ConfigurationFileOptions.cs

```diff
@@ -15,7 +15,7 @@ public class ConfigurationFileOptions
         public int PrintWidth { get; init; } = 100;
         public int TabWidth { get; init; } = 4;
         public bool UseTabs { get; init; }
-        public string EndOfLine { get; init; } = "lf";
+        public string EndOfLine { get; init; } = "auto";
```

Comment:
Why did you make this a string instead of just putting the enum value here?
(Both the Json/Yaml parsers should be able to handle that)